Dockerfile

```diff
@@ -22,7 +22,7 @@ RUN mv bws /usr/local/bin
 RUN rm -r /tmp/
 
 # Install the dependencies
-RUN pip install --no-cache-dir -r requirements.txt
+RUN --mount=type=cache,target=/root/.cache/pip pip install -r requirements.txt
 
 # Copy the application code to the container
 COPY . .
```

Comment:
Include a .Dockerignore for files that don't need to be in the container

Reply:
Speeds up rebuilds that require a pip install by caching the dependencies  

https://pythonspeed.com/articles/docker-cache-pip-downloads/

Reply:
#95